lib/sql-string.js

```diff
@@ -4,6 +4,23 @@ const dataTypes = require('./data-types');
 const util = require('util');
 const _ = require('lodash');
 
+function arrayToList(array, timeZone, dialect, format) {
```

Comment:
Can you cleanup this method, may be use reduce as well

Reply:
Done